admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -0,0 +1,107 @@
+<!--**
+* 2007-2019 PrestaShop SA and Contributors
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Open Software License (OSL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* https://opensource.org/licenses/OSL-3.0
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to https://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2019 PrestaShop SA and Contributors
+* @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+* International Registered Trademark & Property of PrestaShop SA
+*-->
+<template>
+  <div :id="id" class="card-block row">
+    <div class="col-sm">
+      <language-list
+        v-if="languagesCount"
+        :languages="languages"
+        @selectLanguage="selectLanguage"
+        @resetLanguage="resetLanguage"
+      >
+      </language-list>
+
+      <currency-modal
+        :language="selectedLanguage"
+        @close="closeModal"
+        @applyCustomization="applyCustomization"
+      >
+      </currency-modal>
+    </div>
+  </div>
+</template>
+
+<script>
+  import LanguageList from './LanguageList.vue';
+  import CurrencyModal from './CurrencyModal';
+
+  export default {
+    name: 'currency-formatter',
+    data: () => {
```

Comment:
if you use this syntax  you should remove the return statement (eslint airbnb)

Reply:
`data: () => ({selectedLanguage: null}),` is the solution :)

Reply:
Otherwise 
```
data() {
  return {};
}
```

Reply:
Yep; `data: () => ({selectedLanguage: null})` works like a charm